### DISCUSSION TITLE
Criando rede social com Wordpress

### DISCUSSION THREAD

**Discussion by didiraja** [frontendbr/forum#740] [Perguntas e Respostas]
Galera, quero pedir ajuda com um projeto que quero montar pra saciar uma necessidade e aproveitar para estudar código.

## A Meta
Quero criar uma rede social que possa salvar os quadrinhos que li. Tal qual Filmow/Skoo ou Orangotag, só que focada para quadrinhos. Já testei tanto redes sociais para livros ou de quadrinhos e nenhuma me satisfez. Sempre acabavam pecando em usabilidade. A melhorzinha foi o GoodReads mas a marcação em si de leitura eu achei ruim.

## O Problema
Logo vi que era possível montar uma dessas, mesmo que de maneira primitiva, com PHP. Iniciei os estudos e embora esteja avançando, ainda não me adaptei direito á linguagem e percebi que até pro funcionamento mais básico da rede eu precisaria de um conhecimento mais intermediário de PHP e até de MySQL para ela funcionar, por causa das tabelas relacionais.

## A Dúvida
Dito isto, é possível montar uma rede dessas com Wordpress + BuddyPress? Saberia me indicar mais ou menos quais plugins usar?  A parte de exibir "produtos" eu sei que é possível porque muita gente usa WP para fazer portfolio ou site de imóveis. __**A minha duvida seria como implementar esse esquema de marcação nos quadrinhos e associá-los aos usuários**__.

**P.S.** Desculpa ter sido meio prolixo é que estou empolgado com o projeto porque com ele vou poder praticar front-end, wordpress (basicamente o que estou estudando hoje) e futuramente back-end. Ah, e quando digo rede social não é pensando em ser o próximo Facebook não hahah Primeiro quero que funcione pra mim, depois eu espalho pros amigos e AÍ ENTÃO eu vejo de tornar público.

**Comment by klarkc** [comment]
Por experiência própria, para usar algo como uma rede social no Wordpress e personalizar a seu gosto, esteja preparado para se aprofundar na plataforma. O wordpress é muito bom quando você procura coisas simples, mas quanto mais complexo e customizado o sistema, mais ele vai exigir de você comprometimento com o estudo de boas práticas e estudo da plataforma em si.

Uma rede social é um sistema complexo por natureza, ainda mais se você quer usar de uma forma customizada, como parece ser o seu caso. No seu caso (opinião pessoal mesmo) eu iria desenvolver um sistema do zero para fazer isso. A complexidade seria a mesma, e você não precisaria ter de aprender todos os detalhes do plugin e da plataforma wordpress antes de começar a de fato codar o seu MVP.

**Comment by felipeuntill** [comment]
Pelo amor de deus não faça isso...

Você sequer conheçe PHP direito e ainda quer desenvolver em cima de uma solução que vai te deixar engessado em vários aspectos, além de possuir complexidades que você certamente não vai precisar.

Recomendo que você pegue algum projeto open source similar e customize, porém o ideal é que o respectivo não seja feito em cima de uma plataforma que você não domine.
